posts/inside-rust/2020-09-11-stabilizing-intra-doc-links.md

```diff
@@ -0,0 +1,218 @@
+layout: post
+title: "Intra-doc links close to stabilization"
+author: Manish Goregaokar and Joshua Nelson
+team: the rustdoc team <https://www.rust-lang.org/governance/teams/dev-tools#rustdoc>
+---
+
+# Intra-doc links close to stabilization
```

Comment:
I think this is handled by the front matter, no?

```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/37223377/93159161-711c7480-f6c2-11ea-8374-902aa8572a3b.png)